lib/main_containers.pm

```diff
@@ -103,5 +106,5 @@ sub load_container_tests {
         load_host_tests_podman() if ($runtime =~ 'podman');
         load_host_tests_docker() if ($runtime =~ 'docker');
     }
-    loadtest 'console/coredump_collect';
+    loadtest 'console/coredump_collect' unless is_jeos;
```

Comment:
The `coredump_collect` is not necessary for all the other container tests anymore?

Reply:
it was not scheduled before. i tried to keep the schedule intact. thats not completely but still. Check the yaml for comparison

Reply:
If I look at https://openqa.suse.de/tests/7886199#step/coredump_collect/1 I see that it is scheduled for SLES 15-SP3 podman and I expect all other container test runs as well.

Reply:
You're right, this was not there before.